addons/mail/static/src/core/web/chatter.js

```diff
@@ -234,6 +234,9 @@ export class Chatter extends Component {
      * @returns {string}
      */
     get toRecipientsText() {
+        if (this.state.thread.recipients.size === 0) {
+            return _t("No recipient found.");
```

Comment:
Note: this string already exist in the .pot file: https://github.com/odoo-dev/odoo/blob/saas-16.4-mail-chatter-no-recipients-text-did/addons/mail/i18n/mail.pot#L5364

Reply:
The string is already there but it's only registered path is in another file. You should add current file in this POT entry

Reply:
Done !